pkg/fqdn/cache.go

```diff
@@ -71,7 +70,7 @@ type ipEntries map[netip.Addr]*cacheEntry
 // but the key is a DNS name.
 type nameEntries map[string]*cacheEntry
 
-// getIPs returns a sorted list of non-expired unique IPs.
+// getIPs returns an unsorted list of non-expired unique IPs.
```

Comment:
I guess you did an assessment of the users and their assumptions around sorting as part of this proposal to ensure other code wasn't relying on this property?

Reply:
I guess there's mainly two converging points in the stack: `updateIPsForName()` which is updated to do its own sorting, and `mapSelectorsToIPsLocked()` which returns a map, hence is unsorted anyway.

Reply:
Yup, that was my analysis. The goal was to push the sorting "out" as much as possible, so it could be skipped if the caller didn't need sorted results.

It seems small, but it was about 10-15% of the benchmark's CPU time.